Doc/library/sqlite3.rst

```diff
@@ -1205,60 +1216,51 @@ you can let the :mod:`sqlite3` module convert SQLite types to different Python
 types via converters.
 
 
-Using adapters to store additional Python types in SQLite databases
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Using adapters to store custom Python types in SQLite databases
```

Comment:
Just to note—essentially everything before this is [Reference](https://diataxis.fr/reference/), while essentially everything after is really [Guide](https://diataxis.fr/how-to-guides/) material. To help keep things focused and the document size manageable and navigatable, it might be a good idea (in the future, not this PR) to split out the latter into a separate [How-to Guide](https://docs.python.org/3/howto/index.html), retaining the rest in the stdlib reference.

Reply:
I'll create an issue for this.

Reply:
In principle, it doesn't necessarily _have_ to be a separate file, but rather just that "Reference" vs, "How-to/Guide" documentation be cleanly and clearly separated at some level (and equally important, that each appropriate to the need it serves, e.g. making sure that Reference documentation is concise, clear and consistently structured to enable efficient, easy and complete information retrieval, rather than being a lengthy, hard to navigate prose narrative).

In this case, though, since we have distinct "HOWTO" and "Stdlib Reference" docs sections, and the page is very long, moving the latter task-oriented information to a separate "HOWTO" would seem to make more sense.